index.bs

```diff
@@ -502,7 +502,7 @@ The geographic north at a given latitude and longitude can be calculated as foll
         // Compensate for the magnetic declination to get the geographic north.
         console.log('True heading in degrees: ' + (heading + declination));
     });
-    </pre>
+    </xmp>
```

Comment:
Can you fix the indentation here as well?

Reply:
Done. I don't have write access to this repo though.